doc/source/reference/arrays.scalars.rst

```diff
@@ -191,29 +191,31 @@ Inexact types
    This means that variables with equal binary values but whose datatypes are of
    different precisions may display differently::
 
-       >>> f16 = np.float16("0.1")
-       >>> f32 = np.float32(f16)
-       >>> f64 = np.float64(f32)
-       >>> f16 == f32 == f64
-       True
-       >>> f16, f32, f64
-       (0.1, 0.099975586, 0.0999755859375)
-
-   Note that none of these floats hold the exact value :math:`\frac{1}{10}`;
-   ``f16`` prints as ``0.1`` because it is as close to that value as possible,
-   whereas the other types do not as they have more precision and therefore have
-   closer values.
-
-   Conversely, floating-point scalars of different precisions which approximate
-   the same decimal value may compare unequal despite printing identically:
-
-       >>> f16 = np.float16("0.1")
-       >>> f32 = np.float32("0.1")
-       >>> f64 = np.float64("0.1")
-       >>> f16 == f32 == f64
-       False
-       >>> f16, f32, f64
-       (0.1, 0.1, 0.1)
+      >>> import numpy as np
+
+      >>> f16 = np.float16("0.1")
+      >>> f32 = np.float32(f16)
+      >>> f64 = np.float64(f32)
+      >>> f16 == f32 == f64
+      True
+      >>> f16, f32, f64
+      (0.1, 0.099975586, 0.0999755859375)
+
+      Note that none of these floats hold the exact value :math:`\frac{1}{10}`;
+      ``f16`` prints as ``0.1`` because it is as close to that value as possible,
+      whereas the other types do not as they have more precision and therefore have
+      closer values.
+
+      Conversely, floating-point scalars of different precisions which approximate
+      the same decimal value may compare unequal despite printing identically:
+
+      >>> f16 = np.float16("0.1")
+      >>> f32 = np.float32("0.1")
+      >>> f64 = np.float64("0.1")
+      >>> f16 == f32 == f64
+      False
+      >>> f16, f32, f64
+      (0.1, 0.1, 0.1)
```

Comment:
With the indentation change the text now shows up as "raw" text, which I don't think we want. If the only way to avoid the import in the second example is to do this, I'd rather have the extra import.

Reply:
Thanks for noticing! I fixed this based on your other comment in 30301ed484be5a22a0b2e756ea09afc32159cf38.